stl/inc/vector

```diff
@@ -2393,7 +2395,7 @@ public:
 
 #if _ITERATOR_DEBUG_LEVEL != 0
     _CONSTEXPR20 _Difference_type _Total_off(const _Mycont* _Cont) const noexcept {
-        return static_cast<_Difference_type>(_VBITS * (_Myptr - _Cont->_Myvec.data()) + _Myoff);
+        return static_cast<_Difference_type>(_VBITS_DIFF * (_Myptr - _Cont->_Myvec.data()) + _Myoff);

```

Comment:
No change requested: I searched for all mentions of `_VBITS` and it appears that all affected locations are indeed being updated. :heart_eyes_cat: